react/formatAddressToString.ts

```diff
@@ -0,0 +1,43 @@
+import { Address } from 'vtex.checkout-graphql'
+import { CountryRules } from 'vtex.address-context/types'
+import msk from 'msk'
+
+export default function formatAddressToString(
+  address: Address,
+  rules: CountryRules
+) {
+  const displaySpec = rules.display.compact
```

Comment:
how about receiving a display type as a prop? In the future we might want to get the string for a `minimal` display, for example

Reply:
nice! will do

Reply:
done in f92cd9f